www/components/AgendaIcon.js

```diff
@@ -0,0 +1,17 @@
+import CalendarTodayRoundedIcon from '@material-ui/icons/CalendarTodayRounded';
+import AccessAlarmRoundedIcon from '@material-ui/icons/AccessAlarmRounded';
+import styles from "../styles/AgendaIcon.module.css"
+function AgendaIcon() {
+    return (
+        <div className={styles.contain}>
+            <CalendarTodayRoundedIcon style={{fontSize: 35,
+            color: "var(--agendaSecondary)"}}/>
+            <div className={styles.check}>
+                <AccessAlarmRoundedIcon style={{fontSize: 17, 
+                    color: "var(--agendaSecondary)"}}/>
+            </div>
+        </div>
+    )
+}
```

Comment:
Sir, we use two space indents at SocexSolutions.

Reply:
Um sir that is whatever the hell your crappy eslint BS did it as

Reply:
I think it goes off of what you default tab setting is, for me it only allows two spaces.